test/built-ins/BigInt/parseInt/S18.2.5_A1_T7.js

```diff
@@ -0,0 +1,32 @@
+// Copyright 2009 the Sputnik authors.  All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+info: Operator use ToString
+esid: sec-bigint-parseint-string-radix
+description: If Type(value) is Object, evaluate ToPrimitive(value, String)
+---*/
+
+var object = {valueOf: function() {return 1}};
+assert.throws(SyntaxError, () => BigInt.parseInt(object));
+
+var object = {valueOf: function() {return 1}, toString: function() {return 0}};
+assert.sameValue(BigInt.parseInt(object), 0n);
+
+var object = {valueOf: function() {return 1}, toString: function() {return {}}};
+assert.sameValue(BigInt.parseInt(object), 1n);
+
+var object = {valueOf: function() {throw "error"}, toString: function() {return 1}};
+assert.sameValue(BigInt.parseInt(object), 1n);
+
+var object = {toString: function() {return 1}};
+assert.sameValue(BigInt.parseInt(object), 1n);
+
+var object = {valueOf: function() {return {}}, toString: function() {return 1}};
+assert.sameValue(BigInt.parseInt(object), 1n);
+
+var object = {valueOf: function() {return 1}, toString: function() {throw "error"}};
+assert.throws("error", BigInt.parseInt(object));
```

Comment:
Two issues here:
- assert.throws() expects a constructor as its first argument
- and assert.throws() expects a function to execute as its second argument